src/pages/settings/Wallet/ExpensifyCardPage.js

```diff
@@ -74,58 +81,89 @@ function ExpensifyCardPage({
-                                        pan="1234123412341234"
-                                        expiration="11/02/2024"
-                                        cvv="321"
-                                        domain={domain}
-                                    />
-                                ) : (
-                                    <MenuItemWithTopDescription
-                                        description={translate('cardPage.virtualCardNumber')}
-                                        title={CardUtils.maskCard(virtualCard.lastFourPAN)}
-                                        interactive={false}
-                                        titleStyle={styles.walletCardNumber}
-                                        shouldShowRightComponent
-                                        rightComponent={
-                                            <Button
-                                                medium
-                                                text={translate('cardPage.cardDetails.revealDetails')}
-                                                onPress={handleRevealDetails}
-                                            />
-                                        }
-                                    />
-                                )}
+                                <DangerCardSection
+                                    title={translate('cardPage.suspiciousBannerTitle')}
+                                    description={translate('cardPage.suspiciousBannerDescription')}
+                                />
                                 <MenuItemWithTopDescription
-                                    title={translate('cardPage.reportFraud')}
+                                    title={translate('cardPage.reviewTransaction')}
                                     titleStyle={styles.walletCardMenuItem}
-                                    icon={Expensicons.Flag}
+                                    icon={Expensicons.MagnifyingGlass}
+                                    iconFill={theme.icon}
                                     shouldShowRightIcon
-                                    onPress={() => Navigation.navigate(ROUTES.SETTINGS_REPORT_FRAUD.getRoute(domain))}
+                                    brickRoadIndicator="error"
+                                    onPress={() => Link.openOldDotLink('inbox')}
                                 />
                             </>
                         )}
-                        {!_.isEmpty(physicalCard) && (
-                            <MenuItemWithTopDescription
-                                description={translate('cardPage.physicalCardNumber')}
-                                title={CardUtils.maskCard(physicalCard.lastFourPAN)}
-                                interactive={false}
-                                titleStyle={styles.walletCardMenuItem}
-                            />
+
+                        {!isDetectedDomainFraud && (
```

Comment:
Is there a way we could default to this without depending on the fraud variables? Since this the core of this page

Reply:
Due to the requirements, we need to hide this block in case of detected domain fraud